source/includes/_changes.md

```diff
@@ -0,0 +1,144 @@
+# Dataset Changes
+
+The changes of a dataset describe subsequent changes of states that affected the different sections of a dataset. Every action taken on any resource through POST, PUT or DELETE creates a change object that can be retrieved and acted upon.
+
+## The change object
+
+> An example change object
```

Comment:
To ensure homogeneity in all examples, "An" should be deleted. Simply keep "Example change object".